reference_material/data_dictionary

```diff
@@ -0,0 +1,130 @@
+# Current available fields:
+
+## api_endpoint_id:
+The unique identifier for the API endpoint.
+Type: numeric
+## api_id:
+The ID of the API.
+Type: numeric
+
+## api_vendor_id:
+The ID of the API vendor.
+Type: numeric
+
+## api_vendor
+The vendor of the API, e.g "OneLook", "Victoria State Government Australia", "TeejLab"...
+Type: string
+
+## api:
+The name of the API, e.g "Datamuse Dictionary", "DepartmentAPI"...
+Type: string
+
+## category
+The category of the API, e.g "Research & Education", "AI & Data Science"...
+Type: string
+
+## usage_base:
+Free or commercial API.
+Type: string
+
+## sample_response:
+A sample response from the API.
+Type: JSON
+
+## tagset:
+List of keys in the sample response.
+Type: list
+
+## authentication:
```

Comment:
If we are using an ordinal scale,

"None": not safe at all
"Path": risky
"others not listed": okay
"OAuth": strongest